sphinx/ext/autosummary/generate.py

```diff
@@ -79,6 +80,11 @@ def _simple_info(msg):
 def _simple_warn(msg):
     print('WARNING: ' + msg, file=sys.stderr)
 
+def _underline(title, line='='):
+    if '\n' in title:
+        raise ValueError('Can only underline single lines')
+    return title + '\n' + line*len(title)
```

Comment:
Can python modules have names containing these characters, or is that forbidden? If so, someone who knows more about using asian characters should patch this and `apidoc`, and add a test